x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx

```diff
@@ -317,6 +327,7 @@ export const MoreContainer = React.memo<MoreContainerProps>(
                     name: fieldName,
                     value,
                     type: fieldType,
+                    aggregatable: isAggregatable,
```

Comment:
nit: I like to keep the prop name as `isAggregatable` everywhere instead of `aggregatable`.